### PR TITLE
Issue #49 - Adds download csv buttons to report modal

### PR DESCRIPTION
**Pull request by 09Kathy**
This issue does not fix the problem of the peak/hwm tables loading inconsistently. However, the download buttons work consistently so long as the tables are loaded in the Report modal.

### CODE REVIEW

File: f [hunk 1]
```diff
@@ -0,0 +1,3 @@
+  107[m
```
Comment (lprivette) [inline f:1]: Accidental generated file?

Comment (09Kathy) [reply f]: weird. I deleted it

File: build/index.html [hunk 2]
```diff
@@ -8,6 +8,8 @@
     <link rel="shortcut icon" id="favicon" type="image/ico" href="images/favicon-wim.png" />
 
     <script src="scripts/vendor.js"></script>
+    <script src="scripts/json2csv.js"></script>
```
Comment (lprivette) [inline build/index.html:11]: Was this added on purpose? The .js files in the 'scripts' folder get added together in the 'combined.js' file in the build so this shouldn't be necessary

Comment (09Kathy) [reply build/index.html:11]: ah! No. I did add it there and thought I deleted it... looks like I didn't. One sec

**Pull request merged**